integration-tests/test_client_options.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from time import sleep
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_set_ansible_host_info(insights_client):
+    """
+    Test if the ansible-host can be set with satellite
+    Related to https://issues.redhat.com/browse/RHELPLAN-160348,
```

Comment:
I'd use the "canonical" reference here, i.e. https://issues.redhat.com/browse/RHEL-3826